test/unit/test-url-replacements.js

```diff
@@ -595,6 +603,12 @@ describes.sandboxed('UrlReplacements', {}, () => {
     });
   });
 
+  it('should replace PAGE_VIEW_ID_64', () => {
+    return expandUrlAsync('?pid=PAGE_VIEW_ID_64').then(res => {
+      expect(res).to.match(/pid=[a-zA-Z0-9_-]+/);
```

Comment:
nit: can we also add a check on the res length is 64. Thanks

Reply:
Also it would be great if we can add integration test coverage. Please see https://github.com/ampproject/amphtml/blob/master/test/integration/test-amp-analytics.js#L46
Thanks